packages/tempus/src/Tempus_StepperRKBase.hpp

```diff
@@ -30,8 +31,15 @@ class StepperRKBase : virtual public Tempus::Stepper<Scalar>
 
 public:
 
-  virtual int getStageNumber() const { return stageNumber_; }
+  virtual Teuchos::RCP<const RKButcherTableau<Scalar> > getTableau() const
+  { return tableau_; }
+
+  virtual Scalar getOrder()    const{return getTableau()->order();}
+  virtual Scalar getOrderMin() const{return getTableau()->orderMin();}
+  virtual Scalar getOrderMax() const{return getTableau()->orderMax();}
+  virtual int getNumberOfStages() const {return getTableau()->numStages();}
 
+  virtual int getStageNumber() const { return stageNumber_; }
   virtual void setStageNumber(int s) { stageNumber_ = s; }
 
   virtual Teuchos::RCP<Thyra::VectorBase<Scalar> > getStageX() {return stageX_;}
```

Comment:
Should `getStageZ` be moved to the base class as well? and application could simply test if its null to determine if stepper is partitioned or not?

Reply:
I have leaned towards having the get and set functions be in the same class as the member data definition, and would not unnecessarily move the member data definition to a base class.  Does that seem reasonable?

Reply:
I'm ok with that. Thanks